src/infra/files/filesQueryRepo.go

```diff
@@ -157,8 +183,18 @@ func (repo FilesQueryRepo) Get(
 		}
 	}
 
+	shouldReturnContent := false
+	if len(filesToFactory) == 1 {
+		shouldReturnContent = true
+	}
+
 	for _, fileToFactory := range filesToFactory {
-		unixFile, err := repo.unixFileFactory(fileToFactory)
+		filePathIsDir := fileInfo.IsDir() && (fileToFactory.String() == unixFilePath.String())
+		if filePathIsDir {
+			continue
+		}
+
+		unixFile, err := repo.unixFileFactory(fileToFactory, shouldReturnContent)
```

Comment:
Dentro do laço, você tem um contexto micro, portanto, não é preciso variáveis verbosas tais como filePathIsDir. Apenas isDirectory é o suficiente para quem está lendo entenda. Até mesmo a variável fileToFactory poderia ser apenas file. Lembre-se, nomes exportados, tais como nome de VO, métodos, etc, precisam ser mais elaborados para designar sua função. Já num ambiente micro, tais como dentro de um método, switch/if/for etc você já tem um contexto mais focado, onde não é preciso ficar se repetindo. Você só precisaria usar fileToFactory aqui se dentro do mesmo loop tivesse outro tipo de file, entende? Equilibrar a verbosidade, nem muito, nem pouco.